src/Interfaces.ts

```diff
@@ -9,12 +9,16 @@ export enum ClaimType {
   Work = 'Work',
 }
 
+export interface StringToRecursiveObject {
+  [key: string]: string | number | StringToRecursiveObject
```

Comment:
This might also need to be an array of items.

Reply:
Which may mean the key can be a number?

Reply:
I think to support the array we'd do

`[key: string]: string | number | StringToRecursiveObject | ReadonlyArray<string | number | StringToRecursiveObject >`

Can the array be of any of those types @krobi64?

Reply:
Are you saying that the claim itself can be an array?

Reply:
No, I'm saying one of the attributes inside the claim may be an array.

Reply:
Ok then I think @lautarodragan option should work, assuming that the array can be of numbers, strings, and objects?

Reply:
Not the claim, but attributes inside it could (is that it @krobi64?).

I guess something like this would be a valid `claim`:
```js
{
    name: 'A Study in Scarlet',
    author: 'Arthur Conan Doyle',
    tags: ['detective', 'novel', 'detective'],
    dateCreated: '1886-01-01T00:00:00.000Z',
    datePublished: '1887-01-01T00:00:00.000Z',
    stuff: [ { something: true, somethingElse: 42 } ],
    mestedStuff: [ { reallyComplex: { someNestedProp: 42 } } ],
  },
```

Reply:
A simpler alternative would be 

```ts
export interface StringToRecursiveObject {
  readonly [key: string]: number | string | unknown // maybe add array here too
}
```

That would allow programmatic access to first-level children of claim as strings or numbers, but no other level, nor mutation.